pkg/executor/wasm/handler.go

```diff
@@ -83,33 +80,20 @@ func (h *executionHandler) run(ctx context.Context) {
 		h.cancel()
 	}()
 
-	configuredProtocol := strings.ToLower(os.Getenv("OTEL_EXPORTER_OTLP_PROTOCOL"))
-	endpoint := os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT")
-
-	protocol := opentelemetry.GRPC
-	if configuredProtocol != "grpc" {
-		protocol = opentelemetry.HTTP
-	}
-
-	allowInsecure := false
-	if u, err := url.Parse(endpoint); err == nil {
-		host := u.Hostname()
-		endpoint = host + ":" + u.Port()
-		allowInsecure = host == "localhost" || host == "127.0.0.1"
-	}
-
-	conf := &opentelemetry.OTelConfig{
```

Comment:
all of this is now being provided by the `telemetry` package `GetTraceClient` function. there's more detail in [another comment](https://github.com/bacalhau-project/bacalhau/pull/2971/commits/4066ee99b3550d0896e4baaad39cb64d842a0cc2#r1391801813).